lib/models/account-template.js

```diff
@@ -92,7 +90,8 @@ class AccountTemplate {
       tlsCertWebid: userAccount.tlsCertificateWebid,
       tlsCertModulus: userAccount.tlsCertificateModulus,
       tlsCertExponent: userAccount.tlsCertificateExponent,
-      addOwlSameAs
+      addOwlSameAs,
+      serverUri: accountManager.host.serverUri
```

Comment:
If the `accountManager` is only being used to get the `host` instance, maybe just pass in `host` directly? Instead of the full `accountManager`?

Reply:
Fixed